src/os/bluestore/BlueStore.cc

```diff
@@ -3814,11 +3823,18 @@ void BlueStore::ExtentMap::reshard(
 		  dout(20) << __func__ << "    splitting blob, bstart 0x"
```

Comment:
Lets observe that function `blob_escapes_range` does not work as I would expect.
It is not looking if outside range is anything allocated.
As for high end (right side I call it) Blob should be truncated and there should be no mapping, so its OK.
The low end (left side I call it) can have unmapped regions.
I think that some of the problems with split_blob could be avoided if we always split blob that produces non-degenerate results.